AppCenterReactNativeShared/ios/AppCenterReactNativeShared/AppCenterReactNativeShared.m

```diff
@@ -18,16 +19,24 @@ + (NSString *) getAppSecret
     NSString * plistPath = [[NSBundle mainBundle] pathForResource:@"AppCenter-Config" ofType:@"plist"];
     NSDictionary * config = [NSDictionary dictionaryWithContentsOfFile:plistPath];
 
-    appSecret = [config objectForKey:@"AppSecret"];
     // If the AppSecret is not set, we will pass nil to MSAppCenter which will error out, as expected
```

Comment:
We should update this comment. Nil app secret is a valid scenario and MSAppCenter won't error out.

Reply:
We don't need any comment actually now, removed on another branch.